extensions/kogito/deployment/src/main/java/io/quarkus/kogito/deployment/KogitoAssetsProcessor.java

```diff
@@ -108,6 +109,44 @@ public void generatePersistenceInfo(ArchiveRootBuildItem root,
         }
     }
 
+    @BuildStep
+    public List<ReflectiveIgnoreWarningBuildItem> reflectiveDMNREST() {
+        final DotName ORG = DotName.createComponentized(null, "org");
```

Comment:
maybe use https://github.com/quarkusio/quarkus/blob/4676d1326382537ee7989a557c14d70faeb993b1/extensions/kogito/deployment/src/main/java/io/quarkus/kogito/deployment/KogitoAssetsProcessor.java#L355 to create dotNames

Reply:
Indeed!

Reply:
no @mswiderski that is "temp method" seeing the javadoc and the proper way of building dotnames is as currently shown.

Reply:
sorry I did misunderstood which method of reference.

Reply:
@tarilabs can you elaborate more on this why that method is not valid?

Reply:
sorry @mswiderski I misunderstood the feedback.
I implemented this for my case extending to the inner class as per newer commit 
608d49b